factory/triesComponents_test.go

```diff
@@ -110,10 +110,9 @@ func getTriesArgs() factory.TriesComponentsFactoryArgs {
 
 func getCacheCfg() config.CacheConfig {
 	return config.CacheConfig{
-		Type:        "LRU",
-		Size:        10,
-		SizeInBytes: 10,
-		Shards:      1,
+		Type:   "LRU",
+		Size:   10,
+		Shards: 1,
```

Comment:
No, that will yield in an error. Simple LRU is incompatible with the `sizeInBytes` non-zero fields.  Non-zero feilds for `sizeInBytes` will be provided just for `SizeLRU` types of caches. That enforcement was made in order to not have ambiguous configs like
```
[TrieNodesDataPool]
    Size = 50000
    Type = "LRU"
    SizeInBytes = 104857600 #100MB
``` 
Will that be a simple LRU as the type states or the program "knows" that you provided a non-zero `SizeInBytes` field and will switch on its own to SizeLRU? This will result in ~5minutes waste of time each time we read the configs :)

Reply:
ok. sounds fair.